packages/credentials_manager/lib/src/credentials_manager.dart

```diff
@@ -24,11 +25,24 @@ class CredentialsManager {
     }
   }
 
-  Future<String> restoreMnemonic() async {
+  Future<String?> restoreMnemonic() async {
     try {
       String? mnemonicStr = await keyChain.read(accountMnemonic);
-      _log.info("Restored credentials successfully");
-      return mnemonicStr!;
+      _log.info(
+        (mnemonicStr != null)
+            ? "Restored credentials successfully"
+            : "No credentials found in secure storage",
+      );
+      return mnemonicStr;
+    } catch (err) {
+      throw Exception(err.toString());
+    }
+  }
+
+  Future deleteMnemonic() async {
```

Comment:
Are we using deleteMnemonics ?

Reply:
Answered [here](https://github.com/breez/l-breez/pull/120#discussion_r1705333036).